competition/utlis.py

```diff
@@ -0,0 +1,95 @@
+from random import randint
+from django.core.mail import mail_admins, send_mass_mail
+from django.template import loader
+from django.utils.html import strip_tags
+from checkout.models import Entries
+from .models import Competition
+
+def new_competition():
+    """Automatically create new competition when current competition
+    is nearing completion"""
+    Competition().save()
+
+    # Send e-mail to admin to alert of new competition creation
+    html_message = loader.render_to_string(
+        'email_templates/admin_email.html'
+        )
+    message = strip_tags(html_message)
+    mail_admins(
+        'New Competition has been created.',
+        message,
+        fail_silently=True,
+        connection=None,
+        html_message=html_message
+    )
+
+def pick_competition_winner():
+    """Get Competition winner and send out emails"""
```

Comment:
This function attempts to do two things making it very long, make it do one thing and call another function to do the other